tracee-ebpf/tracee/events_decoder.go

```diff
@@ -213,7 +213,11 @@ func readSockaddrFromBuff(buff io.Reader) (map[string]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	res["sa_family"] = helpers.ParseSocketDomain(uint32(family))
+	socketDomainArg, err := helpers.ParseSocketDomainArgument(uint64(family))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Do we really want to return an error in that case?
We can just stringify the family number instead